src/NServiceBus.Core/Sagas/Saga.cs

```diff
@@ -74,7 +76,7 @@ protected void RequestTimeout<TTimeoutMessageType>(DateTime at, Action<TTimeoutM
         /// </summary>
         /// <param name="at"><see cref="DateTime"/> to send timeout <paramref name="timeoutMessage"/>.</param>
         /// <param name="timeoutMessage">The message to send after <paramref name="at"/> is reached.</param>
-        protected void RequestTimeout<TTimeoutMessageType>(DateTime at, TTimeoutMessageType timeoutMessage)
+        protected Task RequestTimeout<TTimeoutMessageType>(DateTime at, TTimeoutMessageType timeoutMessage)
```

Comment:
Good catch. Will obsolete the sync ones and introduce the postfixed ones.
By the way in the end goal we are going to remove the postfix.

-- 
*#NSBcon 2015 • The "All About NServiceBus" conference is back *
*• Learn more <http://go.particular.net/l/27442/2015-08-21/5d4wzp> →*